### PR TITLE
TinyBasic.yaml: Add TinyBasic language

### PR DESCRIPTION
**Pull request by CLiu13**
This adds TinyBasic to the language definitions.

### CODE REVIEW

File: data/Language/TinyBasic.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+identifier: TinyBasic
+full_name: TinyBasic
+versions: 1.0, 2.0
```
Comment (jayvdb) [inline data/Language/TinyBasic.yaml]: discard this.  We dont have versions in the schema yet.  There is a separate task for creating properties ;-)

File: data/Delimiter/tb_comment.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+type: comment
+start: REM
```
Comment (jayvdb) [inline data/Delimiter/tb_comment.yaml]: this deliminter should be called `rem.yaml`.  It is used by many, many early languages ;-)

File: data/Language/TinyBasic.yaml (hunk 1 above)
Comment (CLiu13) [reply data/Language/TinyBasic.yaml]: Gotcha, fixed!

File: data/Delimiter/tb_comment.yaml (hunk 2 above)
Comment (CLiu13) [reply data/Delimiter/tb_comment.yaml]: My bad, fixed!

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/9/commits/d5aec72635915e530186e15e2c1958d52b73ffc7

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward failed! Please fastforward your pull request manually via the command line.

Reason:
```
Command: git merge --ff-only head/define-tinybasic

Exit Code: 128

Cause:

fatal: Not possible to fast-forward, aborting.


```

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**